content/dstu2.md

```diff
@@ -388,11 +390,11 @@ follow these Link header values instead of constructing your own URLs.
       "link": [
         {
           "relation": "self",
-          "url": "https://fhir-open.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Patient?name=Jones&start=0&_count=20"
+          "url": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient?name=Jones&start=0&_count=20"
```

Comment:
This is the syntax for cloud patient search, which isn't present in FHIRPlay.  Madhur/Bradley may be able to give you local patient search examples that will work once that lands on 3/1.  Or, since this example isn't tied to a resource you could use a different one.